src/kube-runtime/src/plugins/ssh/README.md

```diff
@@ -10,9 +10,12 @@ extras:
     - plugin: ssh
       parameters:
         jobssh: boolean
+        sshbarrier:
+          - taskrole
         userssh: 
           type: string
           value: string
 ```
 - jobssh: true to enable job container wise ssh, false to disable.
+- sshbarrier: wait until can ssh to all defined taskrole's job containers. Only valid when jobssh is true.
```

Comment:
please define sshbarrier for the whole job level, instead of the task role level.

Reply:
Now sshbarrier is a boolean to define if barrier is enabled.
Use optional param sshbarriertaskroles to identify specific taskroles that barrier watches to meet advanced requirements.